extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -7,20 +7,20 @@ import { ToolType, ITool } from '../../interfaces';
 import * as nls from 'vscode-nls';
 const localize = nls.loadMessageBundle();
 
-export class MSSQLCtlTool implements ITool {
+export class AzDataTool implements ITool {
```

Comment:
Minor: can we use `AzdataTool`? I don't think "Data" needs capitalizing and it looks weird to read.